components/script/dom/document.rs

```diff
@@ -3473,20 +3473,14 @@ impl Document {
         have_changed
     }
 
-    /// Returns a `Device` suitable for media query evaluation.
+    /// Run the given closure using the Stylo `Device` suitable for media query evaluation.
```

Comment:
I think 3rd person singular is typically used:

```suggestion
    /// Runs the given closure using the Stylo `Device` suitable for media query evaluation.
```

Reply:
Sure. Done.